country-a-service/testing-shared/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/testing/shared/FmkResponseStorage.java

```diff
@@ -15,14 +19,29 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.List;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
 
 /**
  * These methods are concerned with retrieving and serializing responses (ePrescriptions) from FMK
  * so that they can be used as reliable test data for e.g. CDA generation.
  */
 public class FmkResponseStorage {
+    private static final Logger logger = Logger.getLogger(FmkResponseStorage.class.getName());
+    static {
+        ConsoleHandler handler = new ConsoleHandler();
+        handler.setFormatter(new SimpleFormatter());
+        handler.setLevel(Level.INFO);
+
+        logger.addHandler(handler);
+        logger.setUseParentHandlers(false);
+        logger.setLevel(Level.INFO);
+    }
```

Comment:
What is the reasoning behind introducing JUL here?  We're using SLF4J everywhere else.

If it is only to provide logging for the main method, then I don't think we should drag in any logging framework.  System.out.println works fine.  It's only run during development anyway.

Reply:
Changed to use SLF4J (which we have an error from whenever we're running tests, I think. Something about a standard-emitter not being present, or something)